.goreleaser.yaml

```diff
@@ -51,18 +63,22 @@ dockers:
   dockerfile: build/Dockerfile
   build_flag_templates:
   - "--platform=linux/arm64"
+  skip_push: auto
 
 docker_manifests:
 - id: zitadel-latest
   name_template: ghcr.io/zitadel/zitadel:latest
   image_templates:
   - ghcr.io/zitadel/zitadel:{{ .Tag }}-amd64
   - ghcr.io/zitadel/zitadel:{{ .Tag }}-arm64
+  # Skips can and shall be set for individual manifests same as in dockers
+  skip_push: auto
 - id: zitadel-Tag
   name_template: ghcr.io/zitadel/zitadel:{{ .Tag }}
   image_templates:
   - ghcr.io/zitadel/zitadel:{{ .Tag }}-amd64
   - ghcr.io/zitadel/zitadel:{{ .Tag }}-arm64
+  skip_push: auto
```

Comment:
do we need this? shouldn't the tag  always be pushed?

Reply:
if it is not a preview / rc it will be pushed anyway.
both ways are fine with me

Reply:
we could leave that out if you`d prefere to always push